lib/query/querybuilder.js

```diff
@@ -148,6 +148,25 @@ class Builder extends EventEmitter {
     return this;
   }
 
+  // Adds a one or more optimizer hits to the list of "optimizerHints" on the query.
+  optimizerHint() {
+    const hints = helpers.normalizeArr.apply(null, arguments);
```

Comment:
We really should get rid of all the `helpers.normalizeArr.apply(null, arguments);` calls.
Could this be rewritten to be `helpers.normalizeArr(...arguments)` instead? Bonus points if you could avoid using `arguments` at all and just use explicit array as a param.

Reply:
@kibertoad Sure, I just followed the patterns in code arround

Reply:
@kibertoad Changed in 65f488b. I removed variadic arguments, therefore it's now just simple `optimizerHint(hints)`